recipes/gotranx/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "gotranx" %}
+{% set name_underscore = "gotranx" %}
+{% set version = "1.3.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name_underscore }}-{{ version }}.tar.gz"
+  sha256: a00f6a94a6038de21ceee525e8e1870077a4473a4f5e272f6f8696bce4bb28cd
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - "gotranx=gotranx.cli:app"
+  script: python -m pip install -v --no-deps .
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools
+  run:
```

Comment:
is `myokit` missing? It appears in the `pyproject.toml`.

Reply:
The issue is that `myokit` is not on `conda-forge`, which is my I also the reason why I didn't run `pip check`. 
I did update the package to work without `myokit` (see https://github.com/finsberg/gotranx/pull/170), but I think it should be part of the default `pip` dependencies since it is a package that would be needed in most scenarios.

Reply:
It's probably advisable to get `myokit` up on conda-forge. Once that's available, make sure to add it and then add back in the `pip check`.